personal_profile/resources/personal_profiles.py

```diff
@@ -0,0 +1,35 @@
+from django.db import models
+PERSONAL_PROFILES = [
+    [
+        'Microsoft',
+        'Bil Gates',
+        'Im a co-founder of microsoft, and have been the richest guy in the world for a while...',
+        models.DateField(1955, 10, 28),
```

Comment:
In all of those DateField change it to the following: `datetime.date(1955, 10, 28)`, with a date to your choice.
Also replace the current import on the top of the page `from django.db import models` to `import datetime`

Reply:
Solved.